master/static/srv/agent_setup_script.sh.template

```diff
@@ -40,6 +45,7 @@ cat /usr/local/determined/container_startup_script
 echo "#### PRINTING CONTAINER STARTUP SCRIPT END ####"
 
 docker run --init --name determined-agent {{.LogOptions}} \
+    --privileged \
```

Comment:
Would it be possible to grant more fine-grained capabilities to the container? Looking briefly it seems like `--cap-add SYS_BOOT` might work?

Reply:
Hmm, weird. I do think it is unfortunate from a security POV (easier to escalate privileges from agent container / attack host OS), but probably not a huge deal.